includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -758,7 +759,7 @@ public function is_setup_intent_success_creation_redirection() {
 		return ! empty( $_GET['setup_intent_client_secret'] ) && // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 			! empty( $_GET['setup_intent'] ) && // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 			! empty( $_GET['redirect_status'] ) && // phpcs:ignore WordPress.Security.NonceVerification.Recommended
-			'succeeded' === $_GET['redirect_status']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			Payment_Intent_Status::SUCCEEDED === $_GET['redirect_status']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
`redirect_status` may not be related to Payment_Intent_Status?

Reply:
:+1: